app/javascript/components/widgets/weather/moon-phase.jsx

```diff
@@ -2,43 +2,105 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { colors } from '@lib/theme';
 
-const iconRender = (posX, posY, radius, phase) => {
-  const shouldHideFront =
-    phase === 'newMoon' || phase === 'fullMoon' ? 'hidden' : 'visible';
-  const getBackEleFill =
-    phase.includes('Cres') || phase === 'fullMoon'
-      ? colors.white
-      : colors.black;
+const getRearPath = (radius, phase, radiusModifier1) => {
+  const diameter = radius * 2;
+  switch (phase) {
+    case 'newMoon':
+    case 'fullMoon':
+      return ``;
+    case 'waxCres':
+    case 'wanGib':
+      return `M 0 -${radius} A ${diameter * radiusModifier1} ${diameter *
+        radiusModifier1} 0 0,1 0 ${diameter - radius}`;
+    case 'firstQuart':
+    case 'thirdQuart':
+      return `M 0 -${radius} A ${diameter * radiusModifier1} ${diameter *
+        radiusModifier1} 0 0,1 0 ${diameter - radius}`;
+    case 'waxGib':
+    case 'wanCres':
+      return `M 0 -${radius} A ${diameter * radiusModifier1} ${diameter *
+        radiusModifier1} 0 0,0 0 ${diameter - radius}`;
+    default:
+      return ``;
+  }
+};
+
+const getFrontPath = (radius, phase, radiusModifier1, radiusModifier2) => {
+  const diameter = radius * 2;
+  switch (phase) {
+    case 'newMoon':
+    case 'fullMoon':
+      return `M 0 -${radius} A ${diameter * radiusModifier1} ${diameter *
+        radiusModifier1} 0 0,0 0 ${diameter -
+        radius} M 0 -${radius} A ${diameter * radiusModifier1} ${diameter *
+        radiusModifier1} 0 0,1 0 ${diameter - radius}`;
+    case 'waxCres':
+    case 'wanGib':
+      return `M 0 -${radius} A ${diameter * radiusModifier1} ${diameter *
+        radiusModifier1} 0 0,0 0 ${diameter -
+        radius} M 0 -${radius} A ${diameter * radiusModifier2} ${diameter *
+        radiusModifier1} 0 0,1 0 ${diameter - radius}`;
+    case 'firstQuart':
+    case 'thirdQuart':
+      return `M 0 -${radius} A ${diameter * radiusModifier1} ${diameter *
+        radiusModifier1} 0 0,0 0 ${diameter - radius}`;
+    case 'waxGib':
+    case 'wanCres':
+      return `M 0 -${radius} A ${diameter * radiusModifier1} ${diameter *
+        radiusModifier1} 0 0,1 0 ${diameter -
+        radius} M 0 -${radius} A ${diameter * radiusModifier2} ${diameter *
+        radiusModifier1} 0 0,0 0 ${diameter - radius}`;
```

Comment:
Maybe these would be easier to read if we split the commands?

```js
return [
 `M 0 -${radius}`,
 `A ${diameter * radiusModifier1} ${diameter *
        radiusModifier1} 0 0,1 0 ${diameter -
        radius}`
...
].join('')

```
Might be easier to extract pieces that can be reused that way

Reply:
@chewnoill thanks for the review! I've updated the `moon-phase.jsx` file with your requested changes. When you get a chance could you please let me know if there's anything else I should fix?